### PR TITLE
Updated readme

### LINKED ISSUE #53: Add chrome extension link with store icon in repo README

### PR DESCRIPTION
**Pull request by DarshitJain04**
Fixes #53 
Updated readme with the link to the chrome extension and also explained all the features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,42 @@
-# Auto Join Google Meet
-
-## Description
+# Auto Join Chrome Extension
 
 We are all fed up of keeping track of google meet links for our lectures. This chrome extension tries to make your life a bit easy!
+You just have to select the slot and the course and this extension opens the link at the given time ( you can customize the start time ) and also lets you auto join it!
+
+<b>Link : <b/>[Auto Join Chrome Extension](https://chrome.google.com/webstore/detail/auto-join-chrome-extensio/djlpjopjnkojimgiaefgbenbbkkknnln?hl=en)
```
Comment (anshulahuja98) [inline README.md:6]: You should use an icon for the chrome store link
https://www.google.com/search?q=download+on+chrome+store+icon&newwindow=1&sxsrf=ALeKk00ILqRBq3_yl-RGQqN0zORMI2YC_g:1601016924379&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiN0rno3IPsAhXQeisKHZaqDgsQ_AUoAXoECA4QAw&biw=1396&bih=766

**Review by anshulahuja98 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DarshitJain04) [reply README.md:6]: So instead of writing Auto Join Chrome Extension, I'll just add the icon?

Comment (anshulahuja98) [reply README.md:6]: You can maybe just refer how other chrome extension repos do it.

Comment (DarshitJain04) [reply README.md]: # [Auto Join Chrome Extension](https://chrome.google.com/webstore/detail/auto-join-chrome-extensio/djlpjopjnkojimgiaefgbenbbkkknnln?hl=en) output-onlinepngtools (1)

Changed the heading to this. Is it okay?

Comment (anshulahuja98) [reply README.md]: umm, not looking good.
Perhaps use the one which says "Available in the chrome store"
And we can make the picture the URL itself
No need for the text before it

Comment (DarshitJain04) [reply README.md]: @anshulahuja98  done! Please have a look

Comment (anshulahuja98) [reply README.md]: LGTM

**Pull request merged**